Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
Sorry, technically StyleCop warnings. I am not sophisticated, and am sometimes linguistically lazy, so called them "compiler warnings".

I see them in the IDE and the rake build:

![image](https://cloud.githubusercontent.com/assets/3275797/6883641/475b2756-d58d-11e4-968e-8988178985e1.png)

![image](https://cloud.githubusercontent.com/assets/3275797/6883647/6b0936ca-d58d-11e4-8140-48f613f269ec.png)

And as for ReSharper, I do have 

```xml
<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/SPACE_BEFORE_TYPEOF_PARENTHESES/@EntryValue">False</s:Boolean>
```

in my `FakeItEasy.sln.DotSettings.user` file, so that probably explains the difference in behaviour.